pkg/nmi/server/server.go

```diff
@@ -130,10 +140,11 @@ func (s *Server) updateIPTableRules() {
 
 	ticker := time.NewTicker(time.Second * time.Duration(s.IPTableUpdateTimeIntervalInSeconds))
 	defer ticker.Stop()
-
 	// Run once before the waiting on ticker for the rules to take effect
 	// immediately.
 	s.updateIPTableRulesInternal()
+	// delete conntrack entries for prexisting connections to metadata endpoint
+	s.deleteConntrackEntries()
```

Comment:
I don't think it should be under a feature flag since this is fixing a critical bug and not adding a feature, once you created the rules, I believe that most users and you as owner wants all requests to IMDS address to be hijacked, and it's a "bug behaviour" that because of the way TCP works and reusing connection some clients won't get a service by NMI for a long time